src/hotspot/os_cpu/aix_ppc/os_aix_ppc.cpp

```diff
@@ -314,6 +314,14 @@ bool PosixSignals::pd_hotspot_signal_handler(int sig, siginfo_t* info,
           tty->print_cr("trap: %s: %s (SIGTRAP, stop type %d)", msg, detail_msg, stop_type);
         }
 
+        // End life with a fatal error, message and detail message and the context.
+        // Note: no need to do any post-processing here (e.g. signal chaining)
+        va_list va_dummy;
+        VMError::report_and_die(thread, uc, NULL, 0, msg, detail_msg, va_dummy);
+        va_end(va_dummy);
+
+        ShouldNotReachHere();
+
         return false; // Fatal error
```

Comment:
In the two files following on you removed the return statement.  For consistency can you do the same here?

Reply:
Oh, you are right. Will do. 

Thanks, Lois!